Docs/source/developers/testing.rst

```diff
@@ -73,13 +73,25 @@ For easier debugging, it can be convenient to run the tests on your local machin
 
   .. code-block:: sh
 
-       ctest --test-dir build -R laser_acceleration
+       ctest --test-dir build -R "laser_acceleration"
 
 * Run tests except those matching the regular expression ``laser_acceleration``:
 
   .. code-block:: sh
 
-       ctest --test-dir build -E laser_acceleration
+       ctest --test-dir build -E "laser_acceleration"
+
+* Sometimes two or more tests share a large number of input parameters and differ by a small set of options.
+  Such tests typically also share a base string in their names.
+  For example, you can find three different tests named ``test_3d_langmuir_multi``, ``test_3d_langmuir_multi_nodal`` and ``test_3d_langmuir_multi_picmi``.
+  In such a case, if you wish to run the test ``test_3d_langmuir_multi`` only, this can be done again with the ``-R`` regular expression filter via
```

Comment:
```suggestion
  In such a case, if you wish to run the test ``test_3d_langmuir_multi`` only, this can be done again with the ``-R`` regular `expression filter <https://regex101.com>`__ via
```